README.md

```diff
@@ -15,7 +15,7 @@ Here are some ideas to get you started:
 - ⚡ Fun fact: ...
 -->
 
-🔭 I’m currently working on YOUR MOM  
-![your mom](https://cdn.discordapp.com/emojis/770060366477918208.gif?v=1)
+🔭 I’m currently working on THE WORK THAT I'VE BEEN ASSIGNED BY MY BOSS LIKE A GOOD BOY  
+![the work that i've been assigned](https://cdn.discordapp.com/emojis/770060366477918208.gif?v=1)
```

Comment:
Sir, please swap this GIF to something more fitting the context. Perhaps an image of some nice dogs doing things that dogs do?